runtime/autoload/provider/pythonx.vim

```diff
@@ -19,7 +19,11 @@ function! provider#pythonx#Require(host) abort
     call add(args, plugin.path)
   endfor
 
-  return provider#Poll(args, a:host.orig_name, '$NVIM_PYTHON_LOG_FILE')
+  if has('win32')
+    return provider#Poll(args, a:host.orig_name, '$NVIM_PYTHON_LOG_FILE', {'overlapped': v:true})
```

Comment:
I thought I commented this already but maybe it got lot in the ether: `'overlapped': v:true` should just be silently ignored on unix, otherwise every plugin that uses it will need to perform this useless ritual. "overlapped" distinguishes between two modes of operation in win23 that are the same on unix: every unix pipe supports both non-overlapped and overlapped operations already! So specifying this is just redundant, but not an error.